BAM-File-Transfer/BAM
# Receiver Conditional Rendering

# Changes
- Changed the simulation button for the WaitingForBumpSender and WaitingForBumpReceiver component
  - The blue simulation button has been removed. It has been replaced with the 'Fist Bump' image
- Receive button now renders the WaitingForBumpReceiver component instead of immediately making the API fetch call
- Loading spinner/progress is somewhat implemented. It is still buggy due to the conditional rendering (but I wanted to have this code pushed out there to everyone else can continue on bc spinner/progress should not be a blocker)

# How to test
1. Open 2 instances of the app (locahost for now since it is not pushed to main)
2. On the first instance, choose the files (Sender)
3. Press "Send Files"
4. On the second instance, press "Receive Files"
5. On the first instance, press the 'Fist Bump' image to send the files
6. On the second instance, press the 'Fist Bump' to receive the files
7. Loading spinner should show momentarily (a bit buggy at the moment due to conditional rendering)
8. Files should populate a list and be displayed for downloading.
